Source/Core/GeometryPipeline.js

```diff
@@ -1006,8 +1006,6 @@ define([
      * This is used by {@link Primitive} to efficiently render a large amount of static data.
      * </p>
      *
-     * @private
```

Comment:
Only reason I removed it is because the documentation for GeometryPipeline.transformToWorldCoordinates (which is not private) references it.